components/Buttons/button.html

```diff
@@ -57,6 +57,9 @@ <h2>Example of Buttons with icons and floating button</h2>
    add
    </span>
         </div>
+        <div class="link-button">
+            <a href="#">Link Button</a>
+        </div>
         <script src="script.js"></script>
```

Comment:
i did not find any js file,if its not linked to this one remove unnecessary code

Reply:
okkk thanks for pointing out